charts/v2.1/cray-hms-bss/templates/tests/test-functional.yaml

```diff
@@ -12,9 +35,12 @@ metadata:
     app.kubernetes.io/name: "{{ .Release.Name }}-test-functional"
 
 spec:
+  backoffLimit: 0
```

Comment:
This ensures that test job spawns only one pod and returns the result afterwards. Default behavior for Kubernetes jobs is to span up to 6 consecutive pods in case of failure. Each pod is spawned with exponential delay (10s, 20s, etc). This usually takes more then 5 minutes, so `helm test` command times out (if run with with default timeout) instead of returning negative test result.